src/celeritas/user/StepData.hh

```diff
@@ -221,19 +222,19 @@ struct StepPointStateData
  *   collected). The detector ID for inactive threads is always "false".
  */
 template<Ownership W, MemSpace M>
-struct StepStateData
+struct StepSelectionStateData
```

Comment:
I'm not a fan of the updated name since the "selection" is stateless: maybe `StepStateDataImpl` since it's basically "all" the data but we want to have a second copy for performance?

Reply:
I was hoping you'd come up with a better name 😅